Update 4.user-role-description.md (#1000)
vesoft-inc/nebula-docs: docs-2.0/nebula-cloud/4.user-role-description.md

# Cloud Solution roles

After creating a Solution, you need to confirm the role privileges in the Cloud platform. This topic introduces the role privileges in the Cloud Solution.

## Built-in roles

> Nebula Cloud has multiple built-in roles:

Reviewer: ```suggestion
Nebula Graph Cloud has multiple built-in roles:
```

- OWNER  
  - Correspond to the resource manager who can subscribe services and manage resource privileges.
  - Configure `Solution`, create and confirm services, and activate subscription.
  - Start and stop services of the `Solution`.
  - Authorizing the `ROOT` user in the database manually.

>   - Manage the private link of the database and access the database on the Cloud through the private link.

Reviewer 2: ```suggestion
  - Manage private endpoints for access to databases.
```

>   - View resource logs.

Reviewer: ```suggestion
  - View history logs.
```

  - Connect to Nebula Dashboard to monitor specified Solution nodes and services.

- ROOT
  - Correspond to the God of the Nebula Graph database, which is set by the OWNER. For privileges of the God role, see [Built-in role privileges](../7.data-security/1.authentication/3.role-list.md).
  - Manage users with the `USER` role in different spaces in the Cloud platform. When authorizing the `USER` role, you need to specify the role privileges of the Nebula Graph database, including `ADMIN`, `DBA`, `GUEST`, and `USER`. For details, see [Built-in role privileges](../7.data-security/1.authentication/3.role-list.md).

>   - Access the database on the Cloud through the private link.

Reviewer 2: ```suggestion
  - Access the database on the Cloud through a private endpoint.
```

  - Connect to visualized tools, including Nebula Dashboard, Nebula Studio, and Nebula Explorer.

- USER
  - The database users are the Azure subscribers invited by the `ROOT` user in the Cloud platform.
  - Correspond to the `ADMIN`, `DBA`, `GUEST`, or `USER` in the Nebula Graph database.

>   - Access the database on the Cloud through the private link.

Reviewer 2: ```suggestion
  - Access the database on the Cloud through a private endpoint.
```

  - Connect to visualized tools, including Nebula Studio and Nebula Explorer.

## Solutions privileges

On the Solution page, users with different roles will see different sidebars. The following describes the privileges of each role. Among them, Y means that this role can view this page, and N means that it cannot.

|Page|OWNER|ROOT|USER|
|:---|:---|:---|:---|
|Solution Info|Y|Y|Y|
|Applications|Y|Y|Y|

> |Connectivity|Y|N|N|a

Reviewer 3: ```suggestion
|Connectivity|Y|N|N|
```
